documentation/slate/source/includes/apis.md

```diff
@@ -1936,8 +1937,12 @@ For example:
 
 -   overall public transport traffic report on Ile de France coverage
     -   <https://api.navitia.io/v1/coverage/fr-idf/traffic_reports>
+-   overall public transport traffic report on Ile de France coverage with disruptions having tags passed in parameter values
+    -   <https://api.navitia.io/v1/coverage/fr-idf/traffic_reports?tags[]=incidents&tags[]=alerte>
```

Comment:
I would put 'alert' in English (unless it mentions a search giving results) : "alerts" (no final `e`)
And 'incident' singular (no final `s` ) to be consistent